docs/translating.md

```diff
@@ -76,25 +76,111 @@ To ensure we always internationalize our JSON errors messages, the
 Zulip linter (`tools/lint-all`) checks for correct usage.
 
 ## Frontend Translations
-The first step in translating the frontend is to create the translation
-files using `python manage makemessages`. This command will create
-translation files under `static/locale`, the location can be changed by
-passing an argument to the command, however make sure that the location is
-publicly accessible since these files are loaded through XHR in the
-frontend which will only work with publicly accessible resources.
+Zulip uses the [i18next][] library for the frontend translations. To make
+the frontend translations work, the first step is to create the JSON
+formatted [resource][] files using the `python manage makemessages`
+command. This command will create a resource file called `translations.json`
```

Comment:
I'm actually a little confused now, sorry. What is the relationship between, for instance, `locale/es/LC_MESSAGES/django.po` and `static/locale/es/translations.json`? Which one should we be calling "the resource file", and what is a term for the other file?